stock_split_picking/wizards/stock_split_picking.py

```diff
@@ -31,7 +40,7 @@ def action_apply(self):
         return getattr(self, "_apply_%s" % self[:1].mode)()
 
     def _apply_done(self):
-        return self.mapped("picking_ids").split_process()
+        return self.mapped("picking_ids").split_process("quantity_done")
```

Comment:
mapped is useless. It's only need when you process field that do not containt an object
exemple 
- self.mapped("name") : mapped is used as name is a string
- self.picking_ids : no need to use mapped, odoo will do the magic (on old version it was necessary)


```suggestion
        return self.picking_ids.split_process("quantity_done")
```